px-data-grid.html

```diff
@@ -146,6 +146,7 @@
           name="[[column.name]]"
           path="[[column.path]]"
           hidden="[[column.hidden]]"
+          disable-hide="[[column.disableHide]]"
```

Comment:
This prop should declared [in px-data-grid-column](https://github.com/predixdesignsystem/px-data-grid/blob/d4131547797e077b8a7ba529236cdbf0d1a7b2ad/px-data-grid-column.html#L51) and documented [here](https://github.com/predixdesignsystem/px-data-grid/blob/d4131547797e077b8a7ba529236cdbf0d1a7b2ad/px-data-grid.html#L489) and [here](https://github.com/predixdesignsystem/px-data-grid/blob/master/px-data-grid-paginated.html#L339).